frontend/src/util/Utility.ts

```diff
@@ -3,6 +3,9 @@ import { removeUser } from '../api/Room';
 import { disconnect } from '../api/Socket';
 import { errorHandler } from '../api/Error';
 
+// Require validator for identifiers as no types are provided.
+const validateIdentifier = require('valid-identifier');
```

Comment:
When I accepted IntelliJ's suggestion to change it into a default import, I think I got the error you're referring to, but it also provided this second suggestion: "Try `npm install @types/valid-identifier` if it exists". Do you know if this works or not?

Reply:
Yeah unfortunately, that is not actually in the npm registry - I tried that initially but apparently no types are implemented, so I used this new syntax to circumvent that issue.